### PR TITLE
SECURITY.md: add gpg key information

### PR DESCRIPTION
**Pull request by kaspar030**
### Contribution description

Adding the new security@riot-os.org key id.

### CODE REVIEW

File: SECURITY.md
Comment (miri64) [inline SECURITY.md:12]: Where can the public key be downloaded?

Comment (miri64) [reply SECURITY.md:12]: FYI: I opened a PR to add the GPG key to the riot-os.org website. Maybe link to there as well, once it is merged.

Comment (miri64) [reply SECURITY.md:12]: https://github.com/RIOT-OS/riot-os.org/pull/95

Comment (miri64) [reply SECURITY.md:12]: Once merged, it will be at https://www.riot-os.org/assets/keys/security.gpg

File: SECURITY.md
Comment (miri64) [inline SECURITY.md:61]:
~~~suggestion
```
-----BEGIN PGP PUBLIC KEY BLOCK-----

mDMEYY4plhYJKwYBBAHaRw8BAQdALRZ/IJmifuwoSUYTVbKUy9z/m3y0ux6DLMD6
kMs13/+0J1JJT1QtT1Mgc2VjdXJpdHkgPHNlY3VyaXR5QHJpb3Qtb3Mub3JnPoiW
BBMWCAA+FiEERMauRBFy+I00I+gfX3lk0PQjkDMFAmGOKZYCGwMFCQPCZwAFCwkI
BwIGFQoJCAsCBBYCAwECHgECF4AACgkQX3lk0PQjkDMyggEA4lp3RDz+qj1K1veV
AlTh/L90rft7v7hZ9ROWD8mssD8BAI3sc+AESdkNf9lATsuH39/9jIi3CqLDdDxK
TJ9SQwcNiQIzBBABCAAdFiEEZi0l9uxO8DMbg2ZS/bt7xc+P5tsFAmGOKbkACgkQ
/bt7xc+P5tvQLA//fCVUg3B5N5J1gCOSGRlplzFO0DELNl8akecxxFuCUU74Hjyc
NSR4r8lQGhGvAVZLanBTprTWeYXtCuLAFfCwvNitbWXnmXRJawQ4k0TQfUXWNsbf
o84QHtKvxEEwLnubVfz+uATw0eahmU2beh2lEl1PKPnpnvc2q9eM019Ff7RV1poe
UD2ctDZ9yn1GDN6A1E9ejAqxowwPxZfafH6uvPcGnDtvBZ0SB2x+EXvFNDpdaFBm
GEqAOY+wBabk6XV9B5qhu0KeVy0ePHni8JaZQJZX+xo2Nzk14IG66nxBF0zz0qTj
2hntxygrS44lQffqkDl+W/Vyc31/k3vsemLQAaC+/ZV1ULxew1VCmBHKE201+8bS
VmQweoiVBa30HtftOhMtlSi+WHyzwG7KGiD148PJIuQx42Dj/iY0MLHCR5c32giF
tW0xJ6fDkVVC0LkLPfBbMJrKxpX5xyWnWVibWHyAXaI/Sh2oK9uIkvdPFh+rWNe7
Wr3Sokn3oUUE2BVkcOiZO9gMYngx6sDWazYwBMTaDxPISIdQofAPZ3LiW/f12wXq
V39RPXrlR3wDf8frhb8Jfxt1q0KHRbU3Drf8cGjpC42H/HazhH4QugbqfUv3BH0C
zJTYg+S79aDgIqUaW5ASxIi5e20jNKoaRnYg7Y1rYk4ttMtH72XylP1vuCK4OARh
jimWEgorBgEEAZdVAQUBAQdAQCSgXzft+sMtSz1vOEaT/s28u/LVmLjUoGtuAcns
in0DAQgHiH4EGBYIACYWIQRExq5EEXL4jTQj6B9feWTQ9COQMwUCYY4plgIbDAUJ
A8JnAAAKCRBfeWTQ9COQM+6YAP4w2R2qD9yO7ILcDWVyityM7+rmYrpqabbz07kh
CST7fgEAgvz7lVIT4bq7IqhdvcpOERC0Wu9c4AjyX9Y6KB3kIwI=
=3GfE
-----END PGP PUBLIC KEY BLOCK-----
```
~~~

Otherwise, it will be not rendered correctly in HTML format. ;-)

**Review by miri64 — CHANGES_REQUESTED** [review]

File: SECURITY.md
Comment (kaspar030) [reply SECURITY.md:61]: (we're also discussing elsewhere if a link would be best, and where to link to)

Comment (kaspar030) [reply SECURITY.md:62]: done

**Review by miri64 — APPROVED** [review]
Please squash

**Comment by miri64** [comment]
The link to the key (for convenience to add it to your own key ring... `curl https://riot-os.org/assets/keys/security.asc | gpg --import` is a bit easier than doing it via copy-paste) can be added as a follow-up. I want the key information in first, then we can discuss details, if the key should be hosted with the website or with the code here on Github (referring to discussions in the maintainer Matrix chat).

**Pull request merged**
